<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tagesansicht</class>
 <widget class="QDialog" name="Tagesansicht">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MV Boli</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Tagesansicht</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(248, 248, 248);</string>
  </property>
  <widget class="QPushButton" name="neuer_Termin">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>810</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{

  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(0, 88, 0);
}</string>
   </property>
   <property name="text">
    <string>Termin hinzufügen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tag_vor">
   <property name="geometry">
    <rect>
     <x>1400</x>
     <y>280</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
 
  border-radius: 10px;
  padding: 0 8px;
background-color: rgb(0, 111, 0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="tag_zurueck">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>280</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{

  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(0, 88, 0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Date">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>280</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 88, 0);

</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Date Label&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendar">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>460</y>
     <width>311</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>330</y>
     <width>761</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView

{
  border: 2px solid rgb(0, 111, 0);
  border-radius: 10px;
  padding: 0 8px;
background-color: rgb(255, 255,255);
}
</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>491</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QTableView
{
  border: 2px solid rgb(0, 111, 0);
  border-radius: 10px;
  padding: 0 8px;
	background-color: rgb(255, 255, 255);

}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 88, 0);</string>
   </property>
   <property name="text">
    <string>Das steht heute an:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Lernplan">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>810</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{

  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(0, 88, 0);
}</string>
   </property>
   <property name="text">
    <string>Lernplan </string>
   </property>
  </widget>
  <widget class="QPushButton" name="faecher">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>810</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton
{

  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(0, 88, 0);
}</string>
   </property>
   <property name="text">
    <string>Studienfächer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="profil">
   <property name="geometry">
    <rect>
     <x>1770</x>
     <y>200</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
 
  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(0, 88, 0);
}</string>
   </property>
   <property name="text">
    <string>Profil</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 88, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>1921</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(6, 6, 6);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="l1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="l2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="l3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="l4">
   <property name="geometry">
    <rect>
     <x>1420</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="l5">
   <property name="geometry">
    <rect>
     <x>1670</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="l6">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="l7">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ansicht">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>200</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">

QComboBox
{
 border: 2px solid gray;
  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(0, 88, 0);
}</string>
   </property>
   <item>
    <property name="text">
     <string>Tagesansicht</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Wochenansicht</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Monatsansicht</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(0, 88, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>290</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>940</y>
     <width>248</width>
     <height>140</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>1870</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 22, 25);

color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="abmelden">
   <property name="geometry">
    <rect>
     <x>1770</x>
     <y>990</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{

  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(0, 88, 0);
}</string>
   </property>
   <property name="text">
    <string>Abmelden</string>
   </property>
  </widget>
  <widget class="QLabel" name="blackBackround">
   <property name="geometry">
    <rect>
     <x>1799</x>
     <y>89</y>
     <width>66</width>
     <height>66</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(171, 171, 171);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="profilePicture">
   <property name="geometry">
    <rect>
     <x>1800</x>
     <y>90</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>neuer_Termin</zorder>
  <zorder>tag_vor</zorder>
  <zorder>tag_zurueck</zorder>
  <zorder>calendar</zorder>
  <zorder>tableView</zorder>
  <zorder>tableView_2</zorder>
  <zorder>label_2</zorder>
  <zorder>Lernplan</zorder>
  <zorder>faecher</zorder>
  <zorder>profil</zorder>
  <zorder>label</zorder>
  <zorder>line</zorder>
  <zorder>l1</zorder>
  <zorder>l2</zorder>
  <zorder>l3</zorder>
  <zorder>l4</zorder>
  <zorder>l5</zorder>
  <zorder>l6</zorder>
  <zorder>l7</zorder>
  <zorder>Date</zorder>
  <zorder>ansicht</zorder>
  <zorder>username</zorder>
  <zorder>logo</zorder>
  <zorder>exit</zorder>
  <zorder>abmelden</zorder>
  <zorder>blackBackround</zorder>
  <zorder>profilePicture</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
